packages/web/src/scss/components/FileUploader/index.html

```diff
@@ -149,6 +149,66 @@ <h3 id="attachments-multipleWithAccept" hidden>Attachments</h3>
   </div>
   <!-- FileUploader: end -->
 </section>
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">FileUploader Multiple with File Queue Behavior Control</h2>
+
+  <p>
+    If FileUploaderInput has a value for `data-spirit-file-queue-limit` set along with `data-spirit-queue-limit-behavior`,
+    the file upload field will be hidden or disabled. When the value for `data-spirit-queue-limit-behavior`
+    is set to 'hide', the input disappears completely. If you set it to 'disable', the input will only be disabled.
+    After removing a file from the queue, the input will be restored.
+    <br />
```

Comment:
Why not a new paragraph? :) We could avoid <br> and it would be semantically better, IMHO.

```suggestion
    </p>
    <p>
```

Reply:
fixed